pages/linux/beep.md

```diff
@@ -0,0 +1,15 @@
+# Beep
+
+> A utility to beep the pc speaker.
+
+- Send a beep that repeats:
```

Comment:
Just a nitpick: I'd phrase this as something closer to "Sound a beep a given number of times". ("sound", "play" work better than "send" IMO). By the way, how would one just play a single beep? Just `beep`?